js/widgets/widget.backcompat.js

```diff
@@ -42,6 +42,7 @@ if ( $.mobileBackcompat !== false ) {
 		_create: function() {
 			this._setInitialOptions();
 			this._super();
+
```

Comment:
there were other changes this is the results of a rebase that already incorporated them ( apparently minus this one line change )